websites/A/Adkami/presence.ts

```diff
@@ -0,0 +1,100 @@
+    }else if(document.location.pathname.includes("/anime") || (document.location.pathname.includes("/video") && document.location.search.includes("t=0"))){
+      if (document.querySelector("#body > section > div.col-12.col-l-9.fiche-look > div.row.blocshadow > div > h1") !== null) {
+        title = document.querySelector("#body > section > div.col-12.col-l-9.fiche-look > div.row.blocshadow > div > h1");
+        presenceData.details = "Viewing Anime:";
+        presenceData.state = title.innerText;
+             presenceData.buttons = [
+            {
+              label: "View Anime",
+              url: document.URL
+            }
+          ];
+      }else{
+        presenceData.details = "Looking for animes";
+      }
+    }else if(document.location.pathname.includes("/drama") || (document.location.pathname.includes("/video") && document.location.search.includes("t=5"))){
+            if (document.querySelector("#body > section > div.col-12.col-l-9.fiche-look > div.row.blocshadow > div > h1") !== null) {
+        title = document.querySelector("#body > section > div.col-12.col-l-9.fiche-look > div.row.blocshadow > div > h1")
+        presenceData.details = "Viewing Drama:";
+        presenceData.state = title.innerText;
+             presenceData.buttons = [
+            {
+              label: "View Drama",
+              url: document.URL
+            }
+          ];
+      }else{
+        presenceData.details = "Looking for dramas";
+      }
+    }else if(document.location.pathname.includes("/serie") || (document.location.pathname.includes("/video") && document.location.search.includes("t=1"))){
+            if (document.querySelector("#body > section > div.col-12.col-l-9.fiche-look > div.row.blocshadow > div > h1") !== null) {
+        title = document.querySelector("#body > section > div.col-12.col-l-9.fiche-look > div.row.blocshadow > div > h1")
+        presenceData.details = "Viewing Serie:";
+        presenceData.state = title.innerText;
+             presenceData.buttons = [
+            {
+              label: "View Serie",
+
+              url: document.URL
+            }
+          ];
+      }else{
+        presenceData.details = "Looking for series";
+      }
+    }else if(document.location.pathname.includes("/hentai") || (document.location.pathname.includes("/video") && document.location.search.includes("t=4"))){
+            if (document.querySelector("#body > section > div.col-12.col-l-9.fiche-look > div.row.blocshadow > div > h1") !== null) {
+        title = document.querySelector("#body > section > div.col-12.col-l-9.fiche-look > div.row.blocshadow > div > h1")
+        presenceData.details = "Viewing Hentai:";
+        presenceData.state = title.innerText;
+             presenceData.buttons = [
+            {
+              label: "View Hentai",
+
+              url: document.URL
+            }
+          ];
+      }else{
+        presenceData.details = "Looking for juicy contents";

```

Comment:
please add some more appropriate text..

Reply:
I replaced all nsfw things by Anime and adding (-18) and it's more appropriate.